features/step_definitions/assertion_steps.rb

```diff
@@ -1,5 +1,5 @@
-Then("I should see {string}") do |content|
-  expect(page).to have_content content
+Then("I should see {string}") do |String|
+  expect(page).to have_content String
```

Comment:
why did you change it from `content`

Reply:
It should still be content - I made the change back to content.